src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="body">
+    <div class="body__calendar calendar calendar--start-day_sun calendar--month-length_29">
+      <div class="calendar__day calendar__day--1"></div>
```

Comment:
each day should have only one class, use nth-child for choosing days

```suggestion
      <div class="calendar__day"></div>
```